_episodes/08-ci.md

```diff
@@ -1,33 +1,37 @@
 ---
-layout: page
-title: Testing
-subtitle: Continuous Integration
-minutes: 10
+title: Continuous Integration
+teaching: 10
+exercises: 0
+questions:
+- "How can I make sure my changes are compatable with the larger code base?"
+objectives:
+- "Understand how continuous integration speeds software development"
+- "Understand the benefits of continuous integration"
+- "Implement a continuous integration server"
+- "Identify a few options for hosting a continuous integration server"
+keypoints:
+- "Servers exist for automatically running your tests"
+- "Running the tests can be triggered by a GitHub pull request"
+- "CI allows cross platform build testing"
+- "A `.travis.yml` file configures a build on the travis-ci servers"
+- "Many free CI servers are available"
 ---
-> ## Learning Objectives {.objectives}
->
-> *   Understand how continuous integration speeds software development
-> *   Understand the benefits of continuous integration
-> *   Implement a continuous integration server
-> *   Identify a few options for hosting a continuous integration server
-
 
 To make running the tests as easy as possible, many software development teams
 implement a strategy called **continuous integration**.
 As its name implies, continuous integration integrates the test suite into the
 development process. Every time a change is made to the repository, the
 continuous integration system builds and checks that code.
 
-> ## Thought Experiment: Does Your Software Work on Your Colleague's Computer? {.challenge}
+> ## Example
```

Comment:
If this heading has to change, at least retain the content in the title. If you must change the heading for some reason, then this section should begin "Consider the following thought experiment: Does your software work on your colleague's computer?"